examples/hello_world/client/BUILD

```diff
@@ -25,13 +26,31 @@ cc_binary(
     srcs = ["hello_world.cc"],
     deps = [
         "//examples/hello_world/proto:hello_world_cc_grpc",
-        "//examples/utils",
         "//oak/client:application_client",
         "//oak/client:manager_client",
+        "//oak/common:utils",
         "@com_github_grpc_grpc//:grpc++",
         "@com_google_absl//absl/flags:flag",
         "@com_google_absl//absl/flags:parse",
         "@com_google_absl//absl/memory",
         "@com_google_asylo//asylo/util:logging",
     ],
 )
+
+container_image(
```

Comment:
Can you add a TODO(#xxx) here and create an issue to encapsulate this in a `oak_application_server_docker` rule or something like that?

Reply:
Which issue is that? Could you use the format `TODO(#xxx)` where `xxx` is the issue number in GItHub, when possible? So it's easier to see when a PR fixes an issue.